presto-parser/src/main/java/com/facebook/presto/sql/tree/QualifiedName.java

```diff
@@ -45,10 +45,21 @@ public QualifiedName(String name)
 
     public QualifiedName(Iterable<String> parts)
     {
+        this(parts, true);
+    }
+
+    private QualifiedName(Iterable<String> parts, boolean toLower)
```

Comment:
I don't like this, but I'm having trouble coming up with a better way to write this. @martint @dain any ideas?